crt_portal/cts_forms/sorts.py

```diff
@@ -50,7 +52,9 @@ def activity_sort(sort):
             sort_exprs.append(F(sort_item[1::]))
         else:
             sort_exprs.append(F(sort_item))
-
-    sort_exprs.extend([F('id').desc()])
+    if sort_type == 'batch':
+        sort_exprs.extend([F('uuid').desc()])
+    else:
+        sort_exprs.extend([F('id').desc()])
```

Comment:
Django has a `pk` shorthand to reference a model's primary key - I wonder if it'll work here?

```suggestion
    sort_exprs.extend([F('pk').desc()])
```